src/components/TipInput.vue

```diff
@@ -236,5 +252,19 @@ export default {
       }
     }
   }
+
+  .token-tips {
+    display: inline-block;
+    top: -0.15rem;
+    position: relative;
+
+    ::v-deep button {
+      padding: 0;
+    }
+
+    ::v-deep .not-bootstrap-modal-content {
+      min-width: 9rem;
```

Comment:
Does it make sense from the design perspective? Maybe @onvisions just forgot to resize the dropdown to the size of the content.

I'm talking about the difference between:

<img width="246" alt="Screenshot 2020-11-25 at 10 17 29" src="https://user-images.githubusercontent.com/9007851/100194922-98d05a00-2f07-11eb-9439-e5cd53bbc0b8.png">
<img width="246" alt="Screenshot 2020-11-25 at 10 17 45" src="https://user-images.githubusercontent.com/9007851/100194926-9a018700-2f07-11eb-81b3-4d343d9f9725.png">

Stuff like this brings a lot of extra styles that are difficult to maintain.

Reply:
> Tsvetan Kolev
You have posted there 2 identical screenshots as far as I see

@stanislav-slavov so I think we can ignore predefined widths in similar cases.

Reply:
Ok, it is removed from the instance.